### ISSUE TITLE
Extract function is broken as of version 2021.01.16-03.28.20

### ISSUE THREAD

**Issue by bpringe** [clojure-lsp/clojure-lsp#263]
In the latest version as of this writing (2021.01.16-03.28.20), the extract-function refactoring is broken. I verified that with the previous version it worked. I'm not sure if this is a broader problem (haven't checked other refactorings), or if something changed in the way the command should be called (I doubt a breaking change was intended).

In `/tmp/clojure-lsp.out` I see this error when attempting to call the extract-function command:

```
ERROR clojure-lsp.main: #error {
 :cause Wrong number of args (4) passed to: clojure-lsp.refactor.transform/extract-function
 :via
 [{:type clojure.lang.ArityException
   :message Wrong number of args (4) passed to: clojure-lsp.refactor.transform/extract-function
   :at [clojure.lang.AFn throwArity AFn.java 429]}]
 :trace
 [[clojure.lang.AFn throwArity AFn.java 429]
  [clojure.lang.AFn invoke AFn.java 44]
  [clojure.lang.AFn applyToHelper AFn.java 165]
  [clojure.lang.AFn applyTo AFn.java 144]
  [clojure.core$apply invokeStatic core.clj 667]
  [clojure.core$apply invoke core.clj 660]
  [clojure_lsp.feature.refactor$fn__25147 invokeStatic refactor.clj 42]
  [clojure_lsp.feature.refactor$fn__25147 invoke refactor.clj 39]
  [clojure.lang.MultiFn invoke MultiFn.java 229]
  [clojure_lsp.feature.refactor$call_refactor invokeStatic refactor.clj 86]
  [clojure_lsp.feature.refactor$call_refactor invoke refactor.clj 85]
  [clojure_lsp.handlers$refactor invokeStatic handlers.clj 302]
  [clojure_lsp.handlers$refactor invoke handlers.clj 296]
  [clojure_lsp.handlers$execute_command invokeStatic handlers.clj 319]
  [clojure_lsp.handlers$execute_command invoke handlers.clj 310]
  [clojure_lsp.main.LSPWorkspaceService$fn__27891 invoke main.clj 371]
  [clojure.core$binding_conveyor_fn$fn__5754 invoke core.clj 2030]
  [clojure.lang.AFn call AFn.java 18]
  [java.util.concurrent.FutureTask run FutureTask.java 266]
  [java.util.concurrent.ThreadPoolExecutor runWorker ThreadPoolExecutor.java 1149]
  [java.util.concurrent.ThreadPoolExecutor$Worker run ThreadPoolExecutor.java 624]
  [java.lang.Thread run Thread.java 748]]}
```

From trace logs you can see it's called as described in the docs, how it has been called in the past:

```
[Trace - 4:36:38 PM] Sending request 'workspace/executeCommand - (29)'.
Params: {
    "command": "extract-function",
    "arguments": [
        "file:///home/brandon/development/clojure-test/src/core.clj",
        5,
        5,
        "xx"
    ]
}
```

**Comment by ericdallo** [comment]
Thanks for the report, yes, it's broken indeed, I already fixed on #261, but maybe fix on master it's better, I'll try to fix it

**Comment by ericdallo** [comment]
Fixed @bpringe :)
